src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,124 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    public function getBundleData()
+    {
+        $bundles = array();
+        $installedPackages = $this->getInstalledPackages();
+        $enabledBundles = $this->getEnabledBundles();
+
+        foreach ($this->getEnabledBundles() as $bundleName => $bundlePath) {
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (strpos($bundlePath.'/', '/vendor/'.$packageName.'/')) {
```

Comment:
That's true. However, I don't know how common is to not use `vendor/`. I've never seen a project like that. Besides, if that's the case, nothing will break. You simply won't see the versions of the bundles, which is the same as the current situation.